debug.log

```diff
@@ -0,0 +1,4 @@
+[0908/113221.336:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0908/113221.338:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0909/071124.476:ERROR:crash_report_database_win.cc(469)] failed to stat report
+[0909/071124.480:ERROR:crash_report_database_win.cc(469)] failed to stat report
```

Comment:
What is the use of this file?

Reply:
actually, this debug.log appeared while i was rebasing and i tried to fix it and falsely hoped it resolved and but now i made the changes and it is fixed